app/lib/ability.rb

```diff
@@ -10,10 +10,12 @@ def initialize(user)
       can :manage, :all
     elsif user.role_type == SiteCoordinatorRole::TYPE || user.role_type == TeamMemberRole::TYPE || user.role_type == CoalitionLeadRole::TYPE
       can :manage, User, id: user.id
+      can :read, VitaPartner, id: accessible_groups.map(&:id)
```

Comment:
Great job leaning on cancancan here for `:read`. I now see that you went this route because of how `load_vita_partners` in `ClientsController` works.